lib/foreman_theme_satellite/documentation.rb

```diff
@@ -49,6 +49,7 @@ module Documentation
       'Searching' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/searching_and_bookmarking_admin",
       'RolesandPermissions' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/managing_users_and_roles_admin",
       'LDAPAuthentication' => "#{ForemanThemeSatellite.documentation_root}/installing_satellite_server_in_a_connected_network_environment/configuring_external_authentication_satellite#Using_LDAP_satellite",
+      'ExternalAuthentication' => "#{ForemanThemeSatellite.documentation_root}/installing_satellite_server_in_a_connected_network_environment/configuring_external_authentication_satellite",
```

Comment:
Yes, it's roughly the same as the LDAP one, but we have two of them in the UI (as Foreman has those docs more splayed out) and w/o the branding the External one points at the bare "admin satellite" guide which is plain wrong.